rules/aip0192/aip0192.go

```diff
@@ -35,6 +38,22 @@ func AddRules(r lint.RuleRegistry) error {
 	)
 }
 
+// Returns true if this is a deprecated method, false otherwise.
+func isDeprecatedMethod(m *desc.MethodDescriptor) bool {
+	if m.GetMethodOptions() == nil {
+		return false
+	}
+	return *m.GetMethodOptions().Deprecated
+}
+
+// Returns true if this is a deprecated service, false otherwise.
+func isDeprecatedService(s *desc.ServiceDescriptor) bool {
+	if s.GetServiceOptions() == nil {
+		return false
+	}
+	return *s.GetServiceOptions().Deprecated
```

Comment:
Same thing here WRT use of accessor method.

Reply:
Acknowledged.